drivers/include/ethos.h

```diff
@@ -55,6 +55,17 @@ extern "C" {
 #define ETHOS_BAUDRATE STDIO_UART_BAUDRATE
 #endif
 /** @} */
+#else
+#ifndef ETHOS_UART
+#if IS_USED(MODULE_USBUS_CDC_ACM)
+#define ETHOS_UART     UART_DEV(0)
```

Comment:
If I were to pick I'd probably put an error "On USB, there is proper Ethernet, please use that (or explicitly configure your UART device)" in her, but this is probably the more professional default.

Reply:
In this case USB would be used to connect to a PC but ethos (via UART) would be used to connect to another board

Reply:
Ah, so UART_DEV(0) is the physical UART on that board. Makes more sense then, as the "preferred" UART is in that case the high-numbered (from USB) while the low-numbered is the "secondary" one that gets the Ethernet.